src/client/components/Post.jsx

```diff
@@ -55,83 +103,20 @@ const Post = () => {
-                                                                <p className="childDateTime">{child.createdAt}</p>
-                                                                <p className="childContent">{child.content}</p>
-                                                                <div className="childInteract">
-                                                                    <button className="childReply" onClick={handleCommentReply(child)}>Reply</button>
-                                                                    {userData.admin && (
-                                                                        <button className="childDelete">Delete</button>
-                                                                    )}
-                                                                </div>
-                                                            </div>
-                                                        )
-                                                    })
-                                                }
-                                            </div>
-                                        )
-                                    })
-                                }
-                            </div>
-                        )
-                    })
-                }
-                {/*<div className="comment">
-                    <p className="commentUsername">Markus</p>
-                    <p className="commentDateTime">28/02/2022 1:22pm</p>
-                    <p className="commentContent">
-                        Lol ok buddy
-                    </p>
-                    <div className="commentInteract">
-                        <button className="commentReply">Reply</button>
-                        <button className="commentDelete">Delete (Conditional)</button>
-                    </div>
-                </div>
-                <div className="comment">
-                    <p className="commentUsername">Juno</p>
-                    <p className="commentDateTime">28/02/2022 1:21pm</p>
-                    <p className="commentContent">
-                        That's pretty cool
-                    </p>
-                    <div className="commentInteract">
-                        <button className="commentReply">Reply</button>
-                        <button className="commentDelete">Delete (Conditional)</button>
-                    </div>
-                </div>*/}
-            </div>
+
+            {commentArr &&
+                Object.keys(commentArr).map(comment => {
+
+                    return (
+                        <PostComment
+                            comment={commentArr[comment]}
+                            userData={userData}
+                            commentArr={commentArr}
+                            postId={postId}
+                        />
+                    )
+                })
+            }
```

Comment:
What does Object.keys do?

Reply:
It is a way to go through each element of an object like `array.forEach()`. `Object.keys(obj)` targets the _value_ of each key in an object rather than the key's name, and `.map` allows us to mutate that value. Although in this case we are just using map to access the key's contents.